pipeline_steps/common.groovy

```diff
@@ -1074,7 +1087,13 @@ List build_creds_array(String list_of_cred_ids){
     Map creds_bundles = [
       "cloud_creds": ['dev_pubcloud_username',
                       'dev_pubcloud_api_key',
-                      'dev_pubcloud_tenant_id'],
+                      'dev_pubcloud_tenant_id',
+                      'phobos_nodepool_auth_url',
```

Comment:
``Could not find credentials entry with ID 'phobos_nodepool_auth_url'`` is the jenkins failure, indicating that either this is the wrong name or it's not in the credentials store.